### PR TITLE
Fixbug/duplicate name

### PR DESCRIPTION
**Pull request by Feudalman**
## Description

Now we not only check this matter in js on the website, but also check the modifed name which maybe duplicate in web server.

Closes: #125

### CODE REVIEW

File: swanlab/server/api/experiment.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 """
 from datetime import datetime
 from fastapi import APIRouter, Request
-from ..module.resp import SUCCESS_200, NOT_FOUND_404
+from ..module.resp import SUCCESS_200, NOT_FOUND_404, DATA_ERROR_500
```
Comment (SAKURA-CAT) [inline swanlab/server/api/experiment.py]: 这是干啥的

Comment (Feudalman) [reply swanlab/server/api/experiment.py]: 后端检测到实验名重复的时候用了这个返回错误信息

Comment (SAKURA-CAT) [reply swanlab/server/api/experiment.py]: 用422吧

Comment (Feudalman) [reply swanlab/server/api/experiment.py]: 0.o

**Review by SAKURA-CAT — APPROVED** [review]

**Pull request merged**